### DISCUSSION TITLE
name 'res' is not defined

### DISCUSSION THREAD

**Discussion by Raizinnn** [TheLastBen/fast-stable-diffusion#1523] [Q&A]
NameError                                 Traceback (most recent call last)
[<ipython-input-6-89627c654c2d>](https://localhost:8080/#) in <module>
    106   GC=""
    107 GCUNET=GC
--> 108 if res<=640:
    109   GCUNET=""
    110 

NameError: name 'res' is not defined

**Comment by rycossette** [comment]
just got this too

**Comment by meunumerotim333** [comment]
Same error here. Trying to train 768px resolution

**Comment by TheLastBen** [comment]
fixed
